internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -553,3 +554,9 @@ type Ownership struct {
 	// TODO: Polymorphic associations won't enforce foreign key to entities in different tables
 	// we might need to have separated ownership tables for each entity.
 }
+
+type Value struct {
+	EnvID sdktypes.UUID `gorm:"primaryKey;type:uuid;not null"`
```

Comment:
do we need an expiry flag ? 
what about a limit on size ? 
what about a limit on number of entities per env ?

Reply:
all limits should be done in the service layer i think?

re expiry - i agree, but next step?

Reply:
we can do both I guess
but I'm ok with service layer only for now